app/views/shared/summary_cards/_group.html.slim

```diff
@@ -21,13 +21,18 @@
           = humanize_date(resource.published_at)
       .summary-card__metadata
         p
+          span.glyphicon.glyphicon-comment.glyphicon--right
+          span.disqus-comment-count data-disqus-identifier="#{url_for(resource)}" 
+            | 0 Comments
```

Comment:
Yeah that's the line... I'm having some issues too. It works correctly for the first comment but then stops incrementing. Anyway, I'm going to debug it but that's the kind of reasons why I dislike using pre-made/external tools :D